samples/native-config-integration/pom.xml

```diff
@@ -148,22 +148,21 @@
             </build>
         </profile>
         <profile>
-            <id>metadataLocal</id>
+            <id>metadataDefault</id>
```

Comment:
Can you add a new profile (and test) instead of renaming this? Because otherwise we lose testing this case

Reply:
This should now be reverted - I've also removed the test that expects the app to fail when the metadata repo is enabled but not configured, we now use a default repo configuration

Reply:
Added new tests here, this was just a transient change.